pennylane/logging/configuration.py

```diff
@@ -68,21 +76,27 @@ def _configure_logging(config_file):
         )
     with open(os.path.join(_path, config_file), "rb") as f:
         pl_config = tomllib.load(f)
-        logging.config.dictConfig(pl_config)
+        if not config_override:
+            logging.config.dictConfig(pl_config)
+        else:
+            logging.config.dictConfig({**pl_config, **config_override})
 
 
-def enable_logging():
+def enable_logging(config_file: str = "log_config.toml"):
     """
     This method allows to selectively enable logging throughout PennyLane, following the configuration options defined in the ``log_config.toml`` file.
 
     Enabling logging through this method will override any externally defined logging configurations.
 
+    Args:
+        config_file (str): The path to a given log configuration file, parsed as TOML and adhering to the ``logging.config.dictConfig`` end-point. The default argument uses the PennyLane ecosystem log-file configuration.
```

Comment:
So to make sure I understand, the default log file won't be in the user's current working directory?

Reply:
Nope, nor has it ever been. It is a TOML file we ship with PennyLane, that can be overriden by a suer, or edit the in-built one using the newly provided commands.

Reply:
That makes sense, I remember now. Do you think it's worth adding a reminder that `config_path` can be used to find the current location

Reply:
Added a comment and a ref to the config_path function to query the absolute path. Will mark as resolved.